File: examples/AccountVerify.rb
require "hellosign-ruby-sdk"

HelloSign.configure do |config|
  # Configure HTTP basic authorization: api_key
  config.username = "YOUR_API_KEY"

  # or, configure Bearer (JWT) authorization: oauth2
  # config.access_token = "YOUR_ACCESS_TOKEN"
end

account_api = HelloSign::AccountApi.new

data = HelloSign::AccountVerifyRequest.new
data.email_address = "some_user@hellosign.com"

begin
  result = account_api.account_verify(data)
  p result
rescue HelloSign::ApiError => e
  puts "Exception when calling HelloSign API: #{e}"
end

File: examples/AccountVerify.js
import * as HelloSign from "hellosign-sdk";

const accountApi = new HelloSign.AccountApi();

// Configure HTTP basic authorization: api_key
accountApi.username = "YOUR_API_KEY";

// or, configure Bearer (JWT) authorization: oauth2
// accountApi.accessToken = "YOUR_ACCESS_TOKEN";

const data = {
  emailAddress: "some_user@hellosign.com",
};

const result = accountApi.accountVerify(data);
result.then(response => {
  console.log(response.body);
}).catch(error => {
  console.log("Exception when calling HelloSign API:");
  console.log(error.body);
});
